src/bootstrap/builder.rs

```diff
@@ -1730,7 +1730,21 @@ impl<'a> Builder<'a> {
         };
 
         if self.config.print_step_timings && !self.config.dry_run {
-            println!("[TIMING] {:?} -- {}.{:03}", step, dur.as_secs(), dur.subsec_millis());
+            let step_string = format!("{:?}", step);
+            let brace_index = step_string.find("{").unwrap_or_default();
```

Comment:
Please use unwrap_or(0) here, even though it's equivalent -- it is clearer to me when we unwrap explicitly with zero rather than jumping through what a default means in this case.